docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -144,13 +156,16 @@ Install the SkyPilot Helm chart with the following command.
     $ WEB_USERNAME=skypilot
     $ WEB_PASSWORD=yourpassword
     $ AUTH_STRING=$(htpasswd -nb $WEB_USERNAME $WEB_PASSWORD)
-    $ helm upgrade --install skypilot-platform skypilot/skypilot-platform \
+    $ helm upgrade --install skypilot skypilot/skypilot-nightly --devel \
     --namespace $NAMESPACE \
+    --create-namespace \
     --set ingress.auth=$AUTH_STRING
```

Comment:
does `ingress.auth` work here? Seems like we are using either `ingress.authSecret` or `ingress.authCredentials`

Reply:
updated now!